exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,17 @@
+
+_external: true
+title: Hello Rubyric
+description: A simple grading example with Rubyric and Python
+max_submissions: 10
+max_points: 10
+points_to_pass: 6
+min_group_size: 1
+max_group_size: 2
+key: rubyric-python
+lti: Rubyric
+lti_context_id: ''
+lti_resource_link_id: hello-world
+lti_open_in_iframe: true
+lti_aplus_get_and_post: true
+url: /aplus_exercise
```

Comment:
I added a short description

Reply:
I did not test that. But it seems like the group size is not set automatically. I tried the following configuration
```yaml
min_group_size: 1
max_group_size: 4
```
But the result was the following: 🤔 
<sub>Screenshot taken from Rubyric</sub>
![image](https://user-images.githubusercontent.com/10437475/94558037-178e7b00-0268-11eb-9bfd-9467b18d28f2.png)

So it seems like those options are useless. Should I remove them from the example?. In one [comment above](https://github.com/apluslms/course-templates/pull/25#discussion_r496668168), I added a screenshot that shows that the `:min_group_size:` and `:max_group_size:` options cannot be added to the RST directive because it causes compilation errors.

Regarding the other settings, I think the whole module cover most of them, and there is no need for adding more details about them.

Reply:
The group size settings are NOT useless. They are simply not automatically forwarded to Rubyric and we should remind the reader that many settings need to be manually configured in Rubyric (after the exercise has been created in Rubyric either manually or automatically). Note that the group size setting must be defined in BOTH A+ and Rubyric in order to enable group submissions (when Rubyric exercises are integrated into A+). The config.yaml group size settings define them for A+, and once the group size options are added to the RST submit directive, RST can be used too.

Reply:
Although the instructions about groups in Rubyric https://plus.cs.aalto.fi/aplus-manual/experimental/rubyric/04_roles_and_groups/#groups  do not mention the A+ side. I think we should make those changes in a new PR.